projects/SINGE.yaml

```
@@ -0,0 +1,11 @@
+Department: Biostatistics and Medical Informatics
+Description: SINGE uses Granger Causality tests to infer gene regulatory networks from pseudotemporally ordered single-cell transcriptomic data.
+FieldOfScience: Bioinformatics
+ID: '563'
+Name: SINGE
+Organization: University of Wisconsin - Madison
```

Our version doesn't have any spaces around the dash

```suggestion
Organization: University of Wisconsin-Madison
```